### PR TITLE
docs / roadmap 2020

### PR DESCRIPTION
**Pull request by dennisocana**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:

- Updated roadmap for 2020 Q1-Q3

### CODE REVIEW

File: documentation/docs/roadmap.md [hunk 1]
```diff
@@ -37,7 +37,22 @@ The timeline below shows the past milestones we have completed, along with the f
 * Add support for 0x v3
 
 ## Q1 2020
-* Liquidity Mining Launch: Wave 1 (mid-February)
-* More connectors for decentralized and centralized exchanges: KuCoin, Bitfinex, HitBTC, gate.io
+* Liquidity Mining Launch: Open Beta (February 3, 2020)
+* Liquidity Mining Launch: Wave 1 (mid-March)
```
Comment (carlolm) [inline documentation/docs/roadmap.md]: early March

File: documentation/docs/roadmap.md [hunk 2]
```diff
@@ -37,7 +37,22 @@ The timeline below shows the past milestones we have completed, along with the f
...
+* Connector certification program
+
+## Q2 2020
+* Scale liquidity mining
+* Publish real-time liquidity metrics
+* Refactor codebase for easier developing Hummingbot
+* More developer tutorial on building custom strategies
+* Triangular arbitrage strategy
+
+## Q3 2020
```
Comment (carlolm) [inline documentation/docs/roadmap.md]: Change this to 1H 2020

File: documentation/docs/roadmap.md [hunk 3]
```diff
@@ -37,7 +37,22 @@ The timeline below shows the past milestones we have completed, along with the f
...
+* Released connector for [KuCoin](/connectors/kucoin)
+* More connectors for centralized exchanges: [Bitfinex](https://www.bitfinex.com/), [HitBTC](https://hitbtc.com/), [gate.io](https://www.gate.io/)
+* Improve maintenance of exchange connectors
+* Mock connector testing framework
+* Connector certification program
+
+## Q2 2020
+* Scale liquidity mining
+* Publish real-time liquidity metrics
+* Refactor codebase for easier developing Hummingbot
```
Comment (carlolm) [inline documentation/docs/roadmap.md]: easier Hummingbot development

File: documentation/docs/roadmap.md [hunk 4]
```diff
@@ -37,7 +37,22 @@ The timeline below shows the past milestones we have completed, along with the f
...
 ## Q1 2020
-* Liquidity Mining Launch: Wave 1 (mid-February)
-* More connectors for decentralized and centralized exchanges: KuCoin, Bitfinex, HitBTC, gate.io
+* Liquidity Mining Launch: Open Beta (February 3, 2020)
+* Liquidity Mining Launch: Wave 1 (mid-March)
+* Released connector for [KuCoin](/connectors/kucoin)
+* More connectors for centralized exchanges: [Bitfinex](https://www.bitfinex.com/), [HitBTC](https://hitbtc.com/), [gate.io](https://www.gate.io/)
+* Improve maintenance of exchange connectors
+* Mock connector testing framework
+* Connector certification program
```
Comment (carlolm) [inline documentation/docs/roadmap.md]: Connector certification program: move to Q2

File: documentation/docs/roadmap.md (hunk 1 above)
Comment (carlolm) [reply documentation/docs/roadmap.md]: "on Binance and 0x/Bamboo Relay"

File: documentation/docs/roadmap.md [hunk 5]
```diff
@@ -37,7 +37,22 @@ The timeline below shows the past milestones we have completed, along with the f
...
+
+## Q2 2020
```
Comment (carlolm) [inline documentation/docs/roadmap.md]: Add:
* Liquidity mining for [Liquid.com](https://liquid.com) and [KuCoin](https://kucoin.com)

File: documentation/docs/roadmap.md (hunk 1 above)
Comment (dennisocana) [reply documentation/docs/roadmap.md]: Done with requested changes in commit https://github.com/CoinAlpha/hummingbot/pull/1377/commits/796ac078d04a83b3114ac0d72a9990bf4cacd333.

File: documentation/docs/roadmap.md (hunk 2 above)
Comment (dennisocana) [reply documentation/docs/roadmap.md]: Done with requested changes in commit https://github.com/CoinAlpha/hummingbot/pull/1377/commits/796ac078d04a83b3114ac0d72a9990bf4cacd333.

File: documentation/docs/roadmap.md (hunk 3 above)
Comment (dennisocana) [reply documentation/docs/roadmap.md]: Done with requested changes in commit https://github.com/CoinAlpha/hummingbot/pull/1377/commits/796ac078d04a83b3114ac0d72a9990bf4cacd333.

File: documentation/docs/roadmap.md (hunk 4 above)
Comment (dennisocana) [reply documentation/docs/roadmap.md]: Done with requested changes in commit https://github.com/CoinAlpha/hummingbot/pull/1377/commits/796ac078d04a83b3114ac0d72a9990bf4cacd333.

File: documentation/docs/roadmap.md (hunk 5 above)
Comment (dennisocana) [reply documentation/docs/roadmap.md]: Done with requested changes in commit https://github.com/CoinAlpha/hummingbot/pull/1377/commits/796ac078d04a83b3114ac0d72a9990bf4cacd333.

**Pull request merged**